src/components/form/form.types.ts

```diff
@@ -229,4 +229,13 @@ export enum FormLayoutType {
      * Render the form fields using a responsive grid layout
      */
     Grid = 'grid',
+
+    /**
+     * Render the form fields in full-width rows.
+     * Each row can have a leading `icon`, and a field.
+     * `title` and `description` provided by the schema will be placed
+     * on the row itself, and not on the field.
+     * This layout is good for creating UIs for user settings pages.
+     */
+    SettingsRow = 'settingsRow',
```

Comment:
I'm not sure `SettingsRow` is a good name for this, it's too coupled to our feature implementation and could have more use cases than that. I think the name should be more generic

Reply:
agree. what's a good name? 🤔 

Perhaps `rows`? _(reminds a little about the default layout, but perhaps it's still generic enough)_

Reply:
We used `Row`. I think it's good enough

Reply:
@jgroth can I resolve this if you are satisfied ?